Patterns/Dutch/Dutch-DateTime.yaml

```diff
@@ -1,44 +1,44 @@
 ---
 TillRegex: !nestedRegex
-  def: (?<till>\b(to|till|til|until|thru|through)\b|{BaseDateTime.RangeConnectorSymbolRegex})
+  def: (?<till>\b(tot|totdat|gedurende|tijdens|ten tijde van)\b|{BaseDateTime.RangeConnectorSymbolRegex})
   references: [ BaseDateTime.RangeConnectorSymbolRegex ]
-RangeConnectorRegex : !nestedRegex
-  def: (?<and>\b(and|through|to)\b|{BaseDateTime.RangeConnectorSymbolRegex})
+RangeConnectorRegex: !nestedRegex
+  def: (?<and>\b(en|tot en met|t\\m|tot|tot aan)\b|{BaseDateTime.RangeConnectorSymbolRegex})
   references: [ BaseDateTime.RangeConnectorSymbolRegex ]
+ArticleRegex: !simpleRegex
+  def: \b(de|het|een)\b
 RelativeRegex: !simpleRegex
-  def: \b(?<order>dit|volgende|komende|aankomende|aanstaande|deze|huidige|vorig|verleden|voorgaande|vorige|laatste)\b
+  def: \b(?<order>dit|deze|volgend[e]?|komend[e]?|aankomend[e]?|aanstaand[e]?|huidig[e]?|vorig[e]?|verleden|voorgaand[e]?|vorig[e]?|laatste|afgelopen)\b
 StrictRelativeRegex: !simpleRegex
-  def: \b(?<order>dit|volgende|komende|aankomende|aanstaande|deze|huidige|vorig|verleden|voorgaande|vorige|laatste)\b
+  def: \b(?<order>dit|deze|volgend[e]?|komend[e]?|aankomend[e]?|aanstaand[e]?|huidig[e]?|vorig[e]?|verleden|voorgaand[e]?|vorig[e]?|laatste|afgelopen)\b
 UpcomingPrefixRegex: !simpleRegex
-  def: .^
-# TODO: modify below regex according to the counterpart in English
+  def: ((aankomend[e]?|komend[e]?|aanstaande))
 NextPrefixRegex: !nestedRegex
-  def: \b(volgende|komende|aankomende|{UpcomingPrefixRegex})\b
+  def: \b(volgend[e]?|eerstvolgende|{UpcomingPrefixRegex})\b
   references: [ UpcomingPrefixRegex ]
 AfterNextSuffixRegex: !simpleRegex
-  def: \b(na\s+(de\s+)?volgende)\b
+  def: \b(na\s+((de|het)\s+)?volgende)\b
 PastPrefixRegex: !simpleRegex
-  def: .^
-# TODO: modify below regex according to the counterpart in English
+  def: ((deze\s+)?verleden)\b
 PreviousPrefixRegex: !nestedRegex
-  def: (voorgaande|vorig|verleden|vorige|laatste|{PastPrefixRegex})\b
+  def: (voorgaand[e]|vorig[e]?|afgelopen|verleden|laatste|{PastPrefixRegex})\b
```

Comment:
`e` in `voorgaand[e]` should be optional? Like `voorgaand[e]?`

Reply:
yes, this will be updated to be optional. 
"voorgaand" is the uninflected form, "voorgaande" the inflected (and most usual) form.  "voorgaande" as default but "voorgaand"  is also used.